Sources/FoundationEssentials/String/String+Internals.swift

```diff
@@ -35,15 +40,40 @@ extension String {
         // leading slash indicates a rooted path on the drive for the current
         // working directory.
         return try Substring(self.utf8.dropFirst(bLeadingSlash ? 1 : 0)).withCString(encodedAs: UTF16.self) { pwszPath in
+            // lt MAX_PATH not lte MAX_PATH because MAX_PATH includes the null terminator
```

Comment:
This comment is not very clear to me.

Reply:
Removed, since this was refactored (see https://github.com/swiftlang/swift-foundation/pull/1257#discussion_r2052997473)